data/coaches.yml

```diff
@@ -0,0 +1,125 @@
+-
+  name: Silvana Precup
+  description: Secretary
+  image: silvana.jpg
```

Comment:
This could be a kind of a data store for users (coaches) with the image and description fields possibly used for something else in the future. However, if you see no future use, it is easy enough to get rid of them...